docs/src/configuring_zed.md

```diff
@@ -393,93 +393,14 @@ Inlay hints querying consists of two parts: editor (client) and LSP server.
 With the inlay settings above are changed to enable the hints, editor will start to query certain types of hints and react on LSP hint refresh request from the server.
 At this point, the server may or may not return hints depending on its implementation, further configuration might be needed, refer to the corresponding LSP server documentation.
 
-Use `lsp` section for the server configuration, below are some examples for well known servers:
+The following languages have inlay hints preconfigured by Zed:
 
-### Rust
+- [Go](https://docs.zed.dev/languages/go)
+- [Rust](https://docs.zed.dev/languages/rust)
+- [Svelte](https://docs.zed.dev/languages/svelte)
+- [Typescript](https://docs.zed.dev/languages/typescript)
 
-```json
-"lsp": {
-  "rust-analyzer": {
-    "initialization_options": {
-      "inlayHints": {
```

Comment:
This is getting lost due to https://github.com/zed-industries/zed/blob/main/docs/src/languages/rust.md having nothing related to inlay hints at all, let's rather move it there and link https://rust-analyzer.github.io/manual.html#inlay-hints too?

Reply:
I must have missed that. That sounds right to me. I'll update the docs.